python/nav/ipdevpoll/plugins/lldp.py

```diff
@@ -69,11 +72,36 @@ def handle(self):
 
             # Store sentinel to signal that LLDP neighbors have been processed
             shadows.AdjacencyCandidate.sentinel(self.containers, SOURCE)
+            yield self._get_chassis_id(mib)
         else:
             self._logger.debug("LLDP remote table seems unchanged")
 
         stampcheck.save()
 
+    @defer.inlineCallbacks
+    def _get_chassis_id(self, mib):
+        chassis_id_subtype = yield mib.get_next("lldpLocChassisIdSubtype",
+                                                translate_result=True)
+        chassis_id = yield mib.get_next("lldpLocChassisId")
+        if not chassis_id:
+            return
+        chassis_id = lldp_mib.IdSubtypes.get(chassis_id_subtype, chassis_id)
+        info = self.containers.factory((INFO_KEY_LLDP_INFO,
+                                        INFO_VAR_CHASSIS_ID),
+                                       shadows.NetboxInfo)
+        info.value = chassis_id
```

Comment:
This, and the one on line 100, is no good. It will typically save the mac address as a `repr()` of a bytes object, which is of no use to the database view. It needs to be converted to a hexadecimal text representation of the address. These are some addresses seen in my db after applying this:

* `b'<a\x04\xf9\x10@'`
* `b'00:24:A8:4E:85:A0'`